src/index.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Based on Leaflet structure (ver. 1.7)
+ * Based on Leaflet structure (ver. 1.9.3)
```

Comment:
To accept these change, confirm that you have carefully checked all the changes that have occurred in the meantime: https://github.com/Leaflet/Leaflet/compare/v1.7.0...v1.9.3

For example, these are the ones that concern the `src/map/handler/Map.TouchZoom.js` file:


![src/map/handler/Map.TouchZoom.js](https://user-images.githubusercontent.com/9614886/213749026-b9267b9b-8fae-4946-9b89-447094550763.png)